src/request.ts

```diff
@@ -55,7 +55,7 @@ export async function request<T = any, R = any>(
       Authorization: 'Basic ' + Buffer.from(`riot:${credentials?.password}`).toString('base64')
     },
     agent: new https.Agent({
-      rejectUnauthorized: false
+      ca: credentials?.certificate
```

Comment:
As mentioned in the definition for Credentials, we should have ca be appended only if the user wants to use the certicate, otherwise we will default to rejectUnauthorized.